src/modules/dashboard/components/Dialogs/ControlSafeDialog/TransactionTypesSection/TransferFundsSection.tsx

```diff
@@ -111,41 +156,42 @@ const TransferFundsSection = ({
     [safeBalances, selectedTokenAddress],
   );
   const selectedSafe = safes.find(
-    (safe) => safe.contractAddress === values.safe?.profile.walletAddress,
+    (s) => s.contractAddress === safe?.profile.walletAddress,
```

Comment:
I've changed to `inputtedSafe` and set this back to `safe`. Although personally don't feel 's' in this case is confusing because we're calling `safes.find` so I know whatever the variable is refers to an item in "safes", which I would presume to be a safe.